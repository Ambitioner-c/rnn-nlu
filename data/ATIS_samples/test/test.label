车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
车型对比
其他
其他
其他
其他
其他
其他
其他
其他
其他
汽车属性
其他
其他
其他
汽车属性
其他
其他
其他
其他
其他
其他
汽车属性
汽车属性
其他
其他
其他
其他
汽车属性
汽车属性
其他
其他
其他
其他
其他
汽车属性
其他
其他
其他
其他
其他
其他
汽车属性
汽车属性
其他
其他
其他
其他
其他
其他
汽车属性
其他
其他
其他
其他
其他
其他
汽车属性
汽车属性
汽车属性
其他
其他
其他
其他
汽车属性
其他
其他
其他
其他
其他
其他
其他
其他
其他
其他
其他
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
汽车属性
用户观点
用户观点
用户观点
用户观点
汽车属性
用户观点
汽车属性
用户观点
汽车属性
用户观点
汽车属性
用户观点
用户观点
用户观点
汽车属性
用户观点
汽车属性
用户观点
汽车属性
用户观点
汽车属性
汽车属性
用户观点
用户观点
汽车属性
用户观点
用户观点
用户观点
汽车属性
用户观点
用户观点
用户观点
汽车属性
用户观点
汽车属性
用户观点
用户观点
汽车属性
汽车属性
用户观点
汽车属性
汽车属性
汽车属性
用户观点
汽车属性
汽车属性
用户观点
用户观点
用户观点
汽车属性
用户观点
汽车属性
用户观点
汽车属性
汽车属性
汽车属性
用户观点
汽车属性
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
用户观点
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
汽车推荐
