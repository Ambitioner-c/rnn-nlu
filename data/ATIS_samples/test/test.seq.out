 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_name B-attr_slot O B-attr_slot O O B-car_name B-car_model B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-attr_slot O B-compare
 B-car_name B-car_model O O B-car_model B-compare
 B-car_name B-car_model O B-car_name O B-compare
 B-car_name B-car_model O B-car_name B-car_model O O B-compare
 B-car_name B-car_model O B-car_name B-car_model I-car_model O O B-compare
 O B-car_name B-car_model O O B-car_name B-car_model B-attr_slot O B-compare
 B-car_name B-car_model I-car_model B-car_model O B-compare
 B-car_name B-car_model I-car_model O B-car_name B-car_model I-car_model O B-compare
 B-car_name O B-car_model O O O B-car_name O O B-compare O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-attr_slot B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_name B-car_model I-car_model O B-car_name O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-compare O
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_name B-car_model O B-car_name B-car_model I-car_model O B-compare
 B-car_name O B-car_model O B-car_name O O B-compare
 B-car_name B-car_model I-car_model O O B-car_name O O O O B-attr_slot O B-attr_slot O B-attr_slot O B-compare O O O
 B-car_name B-car_model O B-car_name B-car_model B-attr_value O B-compare
 B-car_name B-car_model O B-car_model O B-attr_value
 B-car_name O B-car_name B-car_model O B-compare
 B-car_name B-car_model B-car_name B-car_model B-car_name B-car_model B-car_name B-car_model O O O O O O O O O B-compare O
 B-car_name O B-car_name B-car_model B-recommend O
 O O O B-attr_value O B-car_name O B-car_name B-car_model O O B-recommend O
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 O B-car_name B-car_name B-car_name O B-attr_slot O B-attr_value
 B-car_name O B-car_name O O
 B-car_name O B-car_name O O B-compare O O B-car_name O B-attr_slot O B-car_name O
 B-car_name B-car_model O B-car_name B-car_model O O B-compare
 B-car_name B-car_model O B-car_name B-car_model I-car_model O O B-compare
 B-car_name B-car_model I-car_model O B-car_model I-car_model O B-attr_slot O O B-compare
 O B-car_name O B-car_model I-car_model O O B-compare
 B-car_name B-car_model O O B-car_name O B-compare
 B-car_name B-car_model O B-car_name O O B-compare
 B-car_name B-car_model I-car_model O O B-car_model I-car_model B-compare
 B-car_name B-car_model O B-car_name B-car_model O O B-compare
 O O B-car_name O B-car_model O B-compare O
 O B-car_brand B-car_name B-car_name B-attr_value O B-attr_value O O B-compare
 B-car_name O B-car_model O O B-car_model
 B-car_name B-car_name B-car_name O B-compare O
 O B-car_name O B-car_name O B-compare
 B-car_name B-car_model O B-car_name B-car_model I-car_model O B-compare
 B-car_name B-car_model B-car_model B-compare
 B-car_name B-car_name B-car_name B-car_model O B-car_name B-car_model O O O B-compare
 B-car_name B-car_model O B-car_model O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-attr_slot B-compare
 B-car_name B-car_model O B-car_name B-car_model O O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_brand B-car_name O B-car_name B-car_model O O B-compare
 B-car_name B-car_model O B-car_name B-car_model O B-compare
 B-car_name O B-car_name B-car_model O B-compare
 B-car_name B-car_model B-car_name B-car_model I-car_model O B-car_name B-car_model B-compare O O O
 B-car_brand B-car_name O B-car_name B-car_model O B-recommend
 B-car_name B-car_model O B-car_name B-car_model O O O
 B-car_name B-car_model I-car_model O B-car_name B-car_model O O B-compare
 B-car_name B-car_model B-car_brand B-car_name B-car_model B-recommend O
 B-car_model B-car_name O B-car_name B-car_model O B-compare
 B-car_name B-car_model B-attr_slot O O O B-recommend
 B-car_name B-car_model O B-car_name B-car_model I-car_model O O O O B-attr_slot
 B-car_name B-car_model O O B-attr_value O
 B-car_name O O O O O B-attr_slot
 O O O B-car_name O O B-attr_value O
 O B-car_name B-car_model O O O O O
 B-car_name B-car_model O O O B-attr_slot O O O O O O O O O B-attr_slot O
 B-car_name B-car_model O O O O O
 B-car_name O O O O
 B-car_name B-car_model O O O B-attr_slot
 B-car_name B-car_model B-attr_slot O O O
 O O O B-car_name B-car_model O O O O B-attr_slot
 O O O O B-car_name
 B-car_name B-car_model O O B-attr_slot
 B-car_name O B-car_model B-attr_slot B-car_model B-attr_value I-attr_value B-attr_slot
 B-car_brand B-car_name B-car_model O O B-attr_slot O O O
 O O O O O O O B-car_name O B-attr_slot B-attr_slot B-attr_slot O O O O O O O O O O O O
 O O O B-attr_slot O O
 B-car_name O O O B-attr_value O B-attr_value O
 B-car_name O O O
 B-car_name O O B-car_name O O O O O B-attr_slot
 B-car_name B-car_model O O B-attr_slot
 B-car_name B-car_model B-attr_slot O B-attr_value O
 O O O O B-attr_value O O O
 B-car_name O B-attr_value O
 O B-attr_value O B-car_name B-car_model B-review O
 B-car_name B-car_model O O O O B-attr_slot
 O B-car_brand B-car_name B-car_model O B-attr_slot
 B-car_name O B-attr_value O O
 B-car_name B-car_model O O O B-attr_slot O O B-attr_slot O O B-attr_value O B-car_name B-car_name B-car_name B-car_name B-car_name B-car_name O O O O O O O O
 B-car_name O B-car_name B-car_model O O B-attr_slot O B-attr_slot O O O O B-attr_value O B-attr_value O O B-attr_slot B-review
 O B-car_name O O B-attr_value O
 O B-car_name B-car_model O O B-price
 B-car_name O B-attr_slot B-review
 B-car_name O O O O O O B-car_name O
 O B-car_name B-car_model O O O O O O O
 O B-car_name B-car_model O O O O
 B-car_name B-car_model O B-car_name B-car_model O O O O O O
 O O O O O B-attr_value O O O B-attr_slot O B-attr_slot O O O O
 B-car_name B-car_model O O O B-attr_slot O O O O O O O
 O B-car_name B-car_model B-attr_value B-attr_value B-attr_value B-car_model O B-attr_slot
 B-car_name B-car_model O B-attr_value O O O O B-car_model O B-car_model
 B-car_name O O O B-car_name B-car_model
 O O O O O O B-attr_slot O O O O
 O B-car_name B-car_model O O O O O O
 B-car_name B-car_model O O B-attr_value O O
 B-car_name B-car_model O O B-attr_value O
 B-car_name O O O B-attr_slot
 B-car_name B-car_model B-attr_slot O O
 O O O O O O O O O O O O
 B-car_name B-car_model O O B-attr_slot O
 B-car_name O O O O
 O B-car_model O B-attr_slot O
 O O O O O O O O B-car_brand B-car_name B-car_model I-car_model O O O O O O O O O O
 O O B-car_name B-car_model O B-car_name O B-car_model O O
 O B-car_name B-car_model I-car_model O O B-attr_slot B-attr_slot
 B-car_name B-car_model O O O B-attr_slot O O
 B-car_name B-car_model O O B-attr_slot O O O
 O B-car_brand O O B-car_name B-car_model O B-car_name B-car_model O O
 B-car_brand B-car_name O O O O O O B-attr_slot O O B-attr_slot O O O O O O O
 B-car_name O O B-attr_slot O
 O O O O B-car_name O
 O O O O O B-car_name B-car_model O O O B-attr_slot
 B-car_name O O O B-attr_slot
 B-car_name O B-car_model I-car_model I-car_model O O O O O O O O O O O O O
 B-car_name B-car_model B-car_model O O O O
 O O O B-car_model B-attr_slot O O
 B-car_name O B-attr_value O O O B-attr_slot O O O
 O O B-car_name B-car_model O B-attr_slot O O B-attr_slot O O
 B-car_model O O B-attr_slot O O O
 B-car_name B-car_model O O B-attr_slot
 B-car_name B-car_model O O O B-attr_slot O O O O O O B-attr_slot
 O B-car_name B-car_model O B-attr_slot O O O B-attr_slot O O
 O O O B-car_model B-attr_slot O O O O O O
 B-car_name B-car_model O B-attr_slot O
 O O O B-car_name O B-attr_slot B-attr_slot B-attr_slot O O O
 B-car_name B-car_model I-car_model O B-attr_slot
 B-car_name B-car_model I-car_model O B-attr_slot O B-attr_slot O O
 O B-car_name B-attr_slot
 B-car_name B-car_model O O O O B-attr_slot
 O O B-car_brand B-car_name B-car_model O O O B-attr_slot O O B-attr_slot B-attr_slot O O O B-attr_slot O
 B-car_name O O O B-attr_slot
 B-car_name B-car_model O O B-attr_slot
 O O O B-car_name O B-attr_slot
 B-car_name B-car_model O O
 B-car_name B-car_model I-car_model O B-attr_value I-attr_value O
 B-car_name B-attr_value O O B-attr_slot O O O O O O
 B-car_name B-car_model O O B-attr_slot
 O B-car_name O O B-attr_slot
 O B-car_name O O O O B-attr_slot
 O B-car_name B-car_model O O O B-attr_slot
 B-car_brand B-car_name B-attr_slot O
 B-car_brand B-car_name B-car_model O B-attr_slot O O O
 O B-car_name O O B-attr_slot B-car_model B-attr_slot O
 B-car_brand B-car_name O O B-review
 B-car_name B-car_model O B-review
 B-car_name O O O O O O O O O
 B-car_name B-review
 B-car_name B-car_model O O B-attr_slot O
 O O O O O O O B-review O O O
 B-car_name B-car_model I-car_model B-attr_slot O O O O O
 B-car_name B-car_model O B-attr_slot B-review
 B-car_name B-car_model B-review O B-attr_slot O O O
 O B-car_name B-car_model I-car_model O O O B-review
 B-car_name O B-car_model O O O B-attr_slot
 B-car_name O B-attr_slot
 B-car_name B-car_model B-car_name B-car_model O B-review O
 B-car_name O O B-review B-recommend O O
 O O B-car_name B-car_model I-car_model O O O B-attr_slot O O O
 B-car_name B-car_model I-car_model O O B-review O O O B-review
 B-car_name B-car_model O O O O O O O B-attr_slot B-review O O O
 B-car_name B-car_model B-attr_slot B-review
 B-car_name B-car_model I-car_model O B-attr_slot O O
 B-car_brand B-car_model B-review O O O O O O O
 O O O O O B-attr_slot
 O B-car_name B-car_model I-car_model O B-attr_slot
 O B-car_name B-car_model B-attr_slot B-review
 B-car_name O O B-car_model O O O B-review O O O O O B-review
 B-car_name O O O O B-attr_slot O O
 B-car_name O B-car_model B-review
 O O B-car_name O O O B-review O O O
 O B-car_name B-car_model O B-review
 B-car_name B-car_model O B-car_name B-attr_slot B-attr_slot O
 B-car_brand B-car_name B-car_model O B-attr_slot B-attr_slot O
 B-car_name B-car_model B-car_name O O O O B-attr_value O O B-attr_value
 O O B-car_name B-car_model O O O B-review
 B-car_name B-car_model I-car_model O O O O O O
 B-review B-car_name O O
 O B-car_name O O O O O O B-attr_slot O O B-attr_slot
 B-car_name O O B-attr_slot B-attr_slot B-review
 B-price_from B-price_to O O O B-user_value O O O O O
 B-car_brand B-car_name O B-attr_slot
 B-car_name B-car_model O O B-attr_slot O
 B-car_name O O O O
 O B-car_name O O B-attr_slot O O
 B-car_name B-car_model O B-attr_slot B-review O
 B-car_name B-car_model O B-attr_slot B-review
 B-car_name O B-review
 B-car_name O O O O B-attr_slot
 O B-car_name B-car_model O O O O O O O B-attr_slot O O B-attr_slot
 B-car_name B-car_model I-car_model O O O O B-review O
 B-car_name B-car_model O O O O B-car_model O O O O O O O B-car_name O O O O O O O O O
 B-car_name O O B-user_value O O O
 B-car_name O B-attr_value O O
 O B-car_name B-car_model B-review
 B-car_name B-car_model O O B-attr_slot
 O O B-car_name O O O O O O B-review
 B-car_name B-car_model O O B-attr_slot
 O O B-car_model I-car_model O O O B-car_name O
 B-car_brand B-car_name B-car_model B-attr_slot O B-attr_value O O O O O O O O O
 B-car_name B-car_model O O O O O O O B-review O
 O O O O O O O O O O O B-car_name O O B-attr_slot O O
 B-car_name B-car_model O O O O
 B-car_name B-car_model O O O O B-attr_slot
 B-car_name O B-attr_slot B-review
 O B-car_name B-car_model B-review
 O B-car_name B-car_model B-attr_value B-review
 B-car_name O O B-attr_slot B-review
 B-car_name B-car_model B-attr_slot O B-review O O B-recommend
 B-car_name B-review
 B-car_name B-car_model I-car_model O O O O B-review
 B-car_name B-car_model O O B-review O
 B-car_name B-car_model I-car_model B-attr_slot O O B-review
 B-car_name B-car_model O O O O B-review O
 B-car_name B-car_model O B-review
 B-car_name B-car_model I-car_model B-review
 B-car_name B-car_model O O B-review
 O B-car_name O B-review
 B-car_name B-car_model I-car_model I-car_model B-review
 O O B-car_name B-attr_slot B-review O O O O O O O
 O O B-car_name B-car_model O B-review B-attr_slot O O
 B-car_name B-car_model O O B-attr_slot O B-review O
 B-car_name B-car_model B-attr_slot B-review
 B-car_name O O O O O O B-attr_slot O O O B-attr_slot O B-review
 B-car_name B-car_model I-car_model B-review
 B-car_name O O O O O O O O
 O B-car_name B-car_model I-car_model I-car_model B-review
 O O B-car_name B-car_model O O O B-attr_value O O O O O O O O
 O B-car_name B-car_model O O O O O B-attr_slot
 B-car_name O O B-review
 O B-user_value O I-user_value O O O O B-review
 O O O O O O O
 B-price O O O O O
 O O O B-car_model
 O O O B-attr_slot
 O O O O O O B-attr_slot B-price O O O B-attr_value O B-attr_slot O O
 B-price O O B-attr_value O O O O B-recommend
 B-price O O B-recommend
 B-price_from O B-price_to O O O O O B-compare
 B-car_name B-car_model O B-car_model O B-compare
 B-car_name B-car_model O O B-review
 B-car_name O B-review
 O O B-price O O B-attr_slot O O O
 O B-price_to O O O O O O B-review
 B-recommend O O B-compare O
 O B-car_name B-car_model O B-review
 O B-car_name B-car_model O B-recommend O O
 B-price O O O O O O B-attr_slot B-compare B-recommend O O O
 B-car_name B-car_model O B-car_model B-recommend O O O O O
 O B-car_name B-car_model O B-car_name B-car_model O B-attr_slot O B-compare
 B-price O O O O O B-attr_slot B-compare
 O O O B-car_model O B-attr_slot O
 B-price O O B-attr_slot O B-attr_slot B-compare O O O O O O O O O
 O O B-car_name B-car_model O O O O O O
 O O B-price_from O O B-price_to O O O O O O O O O O B-recommend O B-attr_slot O O O O
 O O B-recommend O O O O B-price_from — B-price_to O O B-car_name model B-car_model O B-car_name B-car_model I-car_model O O O
 B-price_from O B-price_to O O O O B-recommend O
 O O B-recommend O B-price O O O B-car_name B-car_name B-car_model B-car_name O B-car_name B-car_model B-car_name B-car_model O B-car_name B-car_model O O O O
 B-recommend O O O O O O O
 O O O O B-attr_slot O O O O O O O B-attr_slot
 O O O O O O O O B-recommend B-user_value O O
 O O B-car_model B-price_from O B-price_to O
 O B-car_model O B-compare
 O O O B-car_model O O B-user_value O O B-recommend O O O O O O O O O O O O B-price_from B-price_to O O O
 O O B-car_name B-car_model B-attr_slot O O O
 O O B-car_name O O O O
 O O B-price O B-recommend O O O O O O
 B-car_model O
 O B-car_name O O O O
 O B-car_name O B-car_model O O O B-price_to O O O O O O O O
 B-price O O O O B-recommend
 O O O B-recommend
 O B-car_name B-car_model O B-attr_slot O O
 B-car_name O O B-attr_slot B-compare
 B-car_name O O B-attr_slot
 O O B-car_model O O O
 O O O O O O O B-attr_slot O O
 O B-price_from B-price_to O O O O B-compare
 O O O O O O B-user_value O O O B-attr_value B-attr_value B-attr_value O O O O
 B-car_name B-car_model O B-car_name B-car_model B-user_value O O O O
 O O O O
 O O O O O O O B-car_name O O O B-price O O O O O O O B-attr_slot O O O O O O
 O O O O B-car_model B-car_model B-price O
 O O O O O O O O O O B-car_model B-car_name B-car_model O O B-car_name B-car_model B-car_name B-car_model B-car_name B-car_model O
 B-car_name B-car_model B-car_name B-car_model I-car_model B-car_name B-car_model B-car_name I-car_model B-car_brand B-car_model B-car_name B-car_name B-car_model B-car_name B-car_model O O O B-attr_slot O O
 O O O O O B-attr_value O O O O O O O O
 B-price O O B-car_model O B-attr_value O B-attr_value
 B-car_name B-car_model B-price O
 O O B-car_model O O O
 B-car_name O O B-car_name O O B-attr_slot O
